CHANGELOG.md

```diff
@@ -4370,3 +4370,4 @@
 [@masarakki]: https://github.com/masarakki
 [@djudd]: https://github.com/djudd
 [@jemmaissroff]: https://github.com/jemmaissroff
+[@n0h0]: https://github.com/n0h0
```

Comment:
This entry is not required because of metadata changes.

Reply:
thanks.
I will fix and force push.